### PR TITLE
Features/theme basic styling

### PR DESCRIPTION
**Pull request by azkurban**
Includes changes from previous pull request (#338)

### CODE REVIEW

File: src/examples/aggregate-column-basic/example-aggregate-column-basic.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<q-grid caption="Aggregate Column Basic"
+<q-grid class="q-grid-aggregation" caption="Aggregate Column Basic"
```
Comment (klumba12) [inline src/examples/aggregate-column-basic/example-aggregate-column-basic.component.html]: get rid of this class

File: src/theme/basic/index.scss [hunk 2]
```diff
@@ -13,9 +14,33 @@ $pager-primary: #757575;
 $icon-button-disabled: rgba(0,0,0,.26);
 $hidden: transparent;
 
+.q-grid-aggregation {
```
Comment (klumba12) [inline src/theme/basic/index.scss]: you don't need to use specific class just apply for any tfoot, but not for floating rows

File: src/examples/aggregate-column-basic/example-aggregate-column-basic.component.html (hunk 1 above)
Comment (azkurban) [reply src/examples/aggregate-column-basic/example-aggregate-column-basic.component.html]: fixed in my last commits

File: src/theme/basic/templates/body-cell-select.tpl.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<ng-template key='body-cell-select.tpl.html' let-$cell let-$view="$view">
+	    <div>
```
Comment (klumba12) [inline src/theme/basic/templates/body-cell-select.tpl.html]: do we need div here?

File: src/theme/basic/index.scss [hunk 4]
```diff
@@ -4,6 +4,7 @@ $head-sorted: #004559;
 
 $grid-border: #e0e0e0;
 $grid-array-background: #e0e0e0;
+$total-row-border: #e0e0e0;
```
Comment (klumba12) [inline src/theme/basic/index.scss]: lets make similar name convention, for all variables?

**Pull request merged**